packages/lib-classifier/src/components/Classifier/components/FieldGuide/components/FieldGuide/FieldGuideContainer.js

```diff
@@ -13,29 +13,26 @@ function FieldGuideContainer ({
 
   const modalComponent = (size === 'small') ? Modal : MovableModal
   const minHeight = 415
-  const minWidth = 490
+  const minWidth = 550 // includes content + 60px of padding
 
   const [height, setHeight] = useState(minHeight)
 
   function onResize(e, direction, ref, delta, position) {
     if (height !== 'auto' && modalComponent === MovableModal) setHeight('auto')
   }
 
-  const boxHeight = (size === 'small') ? '100%' : `${minHeight}px`
-  const boxWidth = (size === 'small') ? '100%' : `${minWidth}px`
-
```

Comment:
Will this break sizing on phones, where the guide should fill the viewport?

Reply:
Was that something you experienced on small viewport sizes?

The field guide modal fills the viewport when tested on my phone. While developing these changes I ran the code locally and then used my phone browser to test. As a backup, you can now see this branch at https://fe-project-branch.preview.zooniverse.org/projects/nora-dot-eisner/planet-hunters-tess/classify/workflow/11235

Reply:
The field guide looks good. 👍🏻 I was trying to understand the code changes here. 

On small viewports, I can't get past the first tutorial slide to enter a new project. ☹️

https://fe-project-branch.preview.zooniverse.org/projects/kmc35/peoples-contest-digital-archive/classify/workflow/18419

![Screenshot of the People's Contest Digital Archive tutorial on an iPhone. The first step of the tutorial fills the screen, with no way to continue to the second step.](https://user-images.githubusercontent.com/59547/214781745-2fca81ad-dde4-44b2-ac64-7cd41c155d41.jpeg)